.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+
+**Thank your for opening a PR!**
+
+Depending on what your PR does, here are a few things you might want to address in the description:
++ [ ] what are the (breaking) changes that this PR makes?
```

Comment:
do we need this? I almost never see this happen.

Reply:
Changes include new features... And I do think we have breaking changes, and to make matters worse, they are often quite subtle: different initializations, deprecations, changes to default values, re-naming kwargs etc.

Reply:
fair